ntbs-service/Pages/Notifications/Edit/Patient.cshtml.cs

```diff
@@ -35,13 +36,26 @@ public PatientModel(INotificationService service, IPostcodeService postcodeServi
 
             var countries = context.GetAllCountriesAsync().Result;
             Countries = new SelectList(countries, nameof(Country.CountryId), nameof(Country.Name));
-            DomesticOrUnknownCountryIds = countries
+            RenderConditionalCountryFieldIds = countries
                 .Where(c =>
                     c.IsoCode == Models.Countries.UkCode
                     || c.IsoCode == Models.Countries.UnknownCode)
-                .Select(c => c.CountryId.ToString()).ToList();
+                .Select(c => c.CountryId.ToString())
+                .ToList();
 
             Sexes = context.GetAllSexesAsync().Result.ToList();
+
+            var occupations = context.GetAllOccupationsAsync().Result;
```

Comment:
Any reason for `Result` over `await`?

Reply:
Constructors AFAIK can't be async

Reply:
That sounds like a very good reason indeed 😂